scss/layout/_grid.scss

```diff
@@ -119,14 +130,32 @@
     }
   }
 
-  .slab .grid {
-    .grid__item:nth-last-child(-n+2) {
-      border-bottom: none;
-      margin-bottom: 0;
+  .slab {
+    .grid {
+      h3 {
+        line-height: 1.5;
```

Comment:
Yeah I'm not quite sure where to put this because I don't want to affect the other existing `h3`.